app-benchmarks/sysbench/sysbench-1.0_pre20161017.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
```

Comment:
vcs-snapshot was also recommended by @pacho2, but reading the eclass:

# Note: this eclass is no longer needed with the new-style 'archive'
# GitHub URLs. They have sane directory names and stable contents,
# so you should really prefer them.

The example code shows usage with https://github.com/example/${PN}/tarball/v${PV} -> ${P}.tar.gz link which is meant for releases.